packages/fluentui/react-northstar/src/components/Tree/hooks/useTree.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+import { TreeProps } from '../Tree';
+import { flattenTree, getTobeRenderedItemsProps } from './flattenTree';
+import { useTreeActiveState } from './useTreeActiveState';
+import { useTreeBehavior } from './useTreeBehavior';
+
+export function useTree(props: TreeProps) {
+  // build flattened tree from props.items
```

Comment:
Let's avoid comments about _what it does_ and replace them with _why_ if it's needed